src/tests/packages/utils.unmocked.test.ts

```diff
@@ -0,0 +1,63 @@
+import { decodePngMetadata, decodeSvgMetadata } from "../../data/image";
+import { ImportedDataState } from "../../data/types";
+import * as utils from "../../packages/utils";
+import { API } from "../helpers/api";
+
+describe("embedding scene data", () => {
```

Comment:
```suggestion
describe("Test embedding scene data", () => {
```
The block renaming suggestions are mainly for consistency and more descriptive tests - I will add some linters for the test so we can try to follow the similar pattern across all tests.